<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 127);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>277</width>
      <height>226</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
    <property name="renderHints">
     <set>QPainter::TextAntialiasing</set>
    </property>
    <property name="dragMode">
     <enum>QGraphicsView::ScrollHandDrag</enum>
    </property>
    <property name="cacheMode">
     <set>QGraphicsView::CacheNone</set>
    </property>
    <property name="resizeAnchor">
     <enum>QGraphicsView::AnchorUnderMouse</enum>
    </property>
    <property name="optimizationFlags">
     <set>QGraphicsView::DontClipPainter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>121</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="title">
    <string>操作</string>
   </property>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除波形</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出程序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openMyComBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeMyComBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>绘制波形</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停/继续</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkSqlBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>显示数据</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;宋体&quot;;</string>
   </property>
   <property name="title">
    <string>开始时间</string>
   </property>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;宋体&quot;;</string>
   </property>
   <property name="title">
    <string>采样时长</string>
   </property>
   <widget class="QLCDNumber" name="lcdNumberS">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberM">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberH">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;宋体&quot;;</string>
   </property>
   <property name="title">
    <string>温度数据</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);</string>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextBrowserInteraction</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="openLinks">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;宋体&quot;;</string>
   </property>
   <property name="title">
    <string>可燃气体浓度</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);</string>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextBrowserInteraction</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="openLinks">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;宋体&quot;;</string>
   </property>
   <property name="title">
    <string>光强数据</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);</string>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextBrowserInteraction</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="openLinks">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;宋体&quot;;</string>
   </property>
   <property name="title">
    <string>湿度数据</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);</string>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextBrowserInteraction</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="openLinks">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>277</width>
      <height>226</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
    <property name="renderHints">
     <set>QPainter::TextAntialiasing</set>
    </property>
    <property name="dragMode">
     <enum>QGraphicsView::ScrollHandDrag</enum>
    </property>
    <property name="cacheMode">
     <set>QGraphicsView::CacheNone</set>
    </property>
    <property name="resizeAnchor">
     <enum>QGraphicsView::AnchorUnderMouse</enum>
    </property>
    <property name="optimizationFlags">
     <set>QGraphicsView::DontClipPainter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>277</width>
      <height>226</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
    <property name="renderHints">
     <set>QPainter::TextAntialiasing</set>
    </property>
    <property name="dragMode">
     <enum>QGraphicsView::ScrollHandDrag</enum>
    </property>
    <property name="cacheMode">
     <set>QGraphicsView::CacheNone</set>
    </property>
    <property name="resizeAnchor">
     <enum>QGraphicsView::AnchorUnderMouse</enum>
    </property>
    <property name="optimizationFlags">
     <set>QGraphicsView::DontClipPainter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>390</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QGraphicsView" name="graphicsView_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>277</width>
      <height>226</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
    <property name="renderHints">
     <set>QPainter::TextAntialiasing</set>
    </property>
    <property name="dragMode">
     <enum>QGraphicsView::ScrollHandDrag</enum>
    </property>
    <property name="cacheMode">
     <set>QGraphicsView::CacheNone</set>
    </property>
    <property name="resizeAnchor">
     <enum>QGraphicsView::AnchorUnderMouse</enum>
    </property>
    <property name="optimizationFlags">
     <set>QGraphicsView::DontClipPainter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
